<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winMenu</class>
 <widget class="QMainWindow" name="winMenu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Général</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnClients">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStock">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inventaire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStaff">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Personnel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContracts">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contrats</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
